WalletWasabi.Fluent/Views/Shell/DialogsView.axaml

```diff
@@ -68,7 +66,6 @@
               IsCancelEnabled="{Binding CurrentPage.EnableCancel, FallbackValue=False}"
               EnableCancelOnPressed="{Binding CurrentPage.EnableCancelOnPressed, FallbackValue=True}"
               EnableCancelOnEscape="{Binding CurrentPage.EnableCancelOnEscape, FallbackValue=True}"
-              IsEnabled="{Binding $parent.DataContext.IsDialogScreenEnabled}"
```

Comment:
Where is this functionality went?

Reply:
IsEnabled is true by default and the `IsDialogScreenEnabled` flag was also true in the ViewModel and it never ever changed and it will never ever do.

That flag was unnecessary. I removed.